### ISSUE TITLE
Mimic console.log functionality allowing multiple args to be logged with spaces

### ISSUE THREAD

**Issue by drahgon55** [enoex/Bragi-Node#6]
I just switched to using bragi and I had a bunch of log in the format 
`console.log(arg1,arg2,arg3)`

this outputs
`arg1 arg2 arg3`

But doing this
`logger.log("debug",arg1,arg2,arg3)`

will only print 'arg1'

Bragi does not allow this instead it only allows formatters. For example to reproduce the above behavior I have to do this:
l`ogger.log("debug","%s %s %s",arg1,arg2,arg3)`

This requires a log of refactoring. Maybe if there are no formatters in the log use this as the default or add an option to allow this or an option to turn off formatters and use this method as the default.

**Comment by enoex** [comment]
+1 , good point. I will make this the default behavior and add an option to keep the current behavior in the console transport config. Will try to get to this later tonight

**Comment by drahgon55** [comment]
great sorry could not find the enhancement label looked everywhere

**Comment by enoex** [comment]
sorry for the delay - been a hectic past couple weeks, getting to this soon!

**Comment by drahgon55** [comment]
sounds good. thx

**Comment by enoex** [comment]
Sorry again for the delay. I added a config option, `hideUnformattedParameters`, which can be set on the logger's options object. By default now, `logger.log("debug",arg1,arg2,arg3)` will include the args in the string; so this should produce the requested behavior. I added a couple examples in example-simple.js to demonstrate the functionality for disabling it: https://github.com/enoex/Bragi-Node/blob/master/examples/example-simple.js#L74
